supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/request/AbortRequestHandler.scala

```diff
@@ -4,20 +4,20 @@ import akka.actor.ActorRef
 import com.google.api.client.googleapis.batch.BatchRequest
 import com.google.api.client.googleapis.json.GoogleJsonError
 import com.google.api.client.http.HttpHeaders
+import com.typesafe.scalalogging.LazyLogging
 import cromwell.backend.google.pipelines.common.api.PipelinesApiRequestManager._
-import cromwell.backend.google.pipelines.common.api.clients.PipelinesApiAbortClient.{PAPIAbortRequestSuccessful, PAPIOperationAlreadyCancelled, PAPIOperationHasAlreadyFinished}
+import cromwell.backend.google.pipelines.common.api.clients.PipelinesApiAbortClient.{PAPIAbortRequestSuccessful, PAPIOperationHasAlreadyFinished}
 import cromwell.cloudsupport.gcp.auth.GoogleAuthMode
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success, Try}
 
-trait AbortRequestHandler { this: RequestHandler =>
+trait AbortRequestHandler extends LazyLogging { this: RequestHandler =>
   protected def handleGoogleError(abortQuery: PAPIAbortRequest, pollingManager: ActorRef, e: GoogleJsonError, responseHeaders: HttpHeaders): Try[Unit] = {
-    // No need to fail the request if the job was already cancelled, we're all good
-    if (Option(e.getCode).contains(400) && Option(e.getMessage).contains("Operation has already been canceled")) {
-      abortQuery.requester ! PAPIOperationAlreadyCancelled(abortQuery.jobId.jobId)
-      Success(())
-    } else if (Option(e.getCode).contains(400) && Option(e.getMessage).contains("Operation has already finished")) {
+    // No need to fail the request if the job already terminated, we're all good
+    // As seen in `cromwell.backend.google.pipelines.common.api.clients.PipelinesApiAbortClient` the difference between "finished" and "cancelled" only affects logging
+    if (Option(e.getCode).contains(400)) {
+      logger.info(s"Tried to abort job ${abortQuery.jobId.jobId} in workflow ${abortQuery.workflowId} but PAPI reports it is no longer running. Marking as finished. Message: ${e.getMessage}")
```

Comment:
So, we decided to not do re-check actual job status after we get 400? Or this will be improved later through separate ticket?

Reply:
Separate ticket I think - this is the "make the current problem go away" version

Reply:
I didn't quite follow that discussion.

Is there a case where a 400 can mean a temporary rather than a permanent failure?

Reply:
In this case I'd make log message sounds less confident, like "... but according to PAPI response it is most likely no longer running". But anyway LGTM

Reply:
> I didn't quite follow that discussion.
> 
> Is there a case where a 400 can mean a temporary rather than a permanent failure?

Well, the only thing it now tells us is that some "condition is not met". In this case we are almost sure that it means that current job status is already terminal. But I think we cannot be sure that there are no other cases when we'd get the same error code and message